packages/teleport-plugin-vue-base-component/src/index.ts

```diff
@@ -28,13 +29,19 @@ export const createPlugin: ComponentPluginFactory<VueComponentConfig> = (config)
     const dataObject: Record<string, any> = {}
     const methodsObject: Record<string, any> = {}
 
-    const templateContent = generateNodeSyntax(uidl.node, {
+    let templateContent = generateNodeSyntax(uidl.node, {
       templateLookup,
       dependencies,
       dataObject,
       methodsObject,
     })
 
+    if (typeof templateContent === 'string') {
```

Comment:
For every node we are adding a ``span`` as a wrapper expect in the case of static, where we are returning the string content itself. I tried adding in ``generateNodeSyntax`` itself, but there are edge cases where it will add two **span** tags. So added here, this will even work as a fall back when the template is returning a string, it will wrap in span so, in HAST generation, the pipeline won't break;

Reply:
Although the tests include a staticNode, the code is not covered. I think we should investigate why.